tests/unit/test_xblock.py

```diff
@@ -48,57 +50,16 @@ def test_course(self):
         self.assertEqual(self.block.course, mock_course)
         self.runtime_mock.modulestore.get_course.assert_called_once_with(self.DEFAULT_COURSE_ID)
 
-    @ddt.data(
-        ([], []),
-        (["dalite-ng:QWE:ASD"], []),  # non dalite-xblock pasport ignored
-        # correctly parses values for dalite-xblock
-        (
-            ["(dalite-xblock)dalite-ng;http://qwe.asd/;QWE;ASD"],
-            [DaliteLtiPassport("dalite-ng", "http://qwe.asd/", "QWE", "ASD")]
-        ),
-        # and can ignore non dalite-xblock passport
-        (
-            ["dalite-ng:QWE:ASD", "(dalite-xblock)dalite-ng-2;https://other.url:8080;KEY;SECRET"],
-            [DaliteLtiPassport("dalite-ng-2", "https://other.url:8080", "KEY", "SECRET")]
-        ),
-        # two dalite-xblock passports
-        (
-            [
-                "dalite-ng:QWE:ASD",
-                "(dalite-xblock)dalite-ng-3;https://other.url:8080;KEY;SECRET",
-                "(dalite-xblock)dalite-ng-4;;;",
-            ],
-            [
-                DaliteLtiPassport("dalite-ng-3", "https://other.url:8080", "KEY", "SECRET"),
-                DaliteLtiPassport("dalite-ng-4", "", "", ""),  # technically, this is valid
-            ]
-        ),
-    )
-    @ddt.unpack
-    def test_dalite_xblock_lti_passports(self, lti_passports, dalite_passports):
+    def test_dalite_xblock_lti_passports(self):
         """Test dalite_xblock_lti_passports property."""
-        self.mock_course.lti_passports = lti_passports
-
-        self.assertEqual(self.block.dalite_xblock_lti_passports, dalite_passports)
-
-    def test_malformed_dalite_xblock_lti_passport(self):
-        """Test malformed LTI passports does not crash XBlock, and record warnings."""
-        lti_passports = [
-            "(dalite-xblock)missing-component;KEY;SECRET",
-            "(dalite-xblock)extra-component;https://other.url:8080;KEY;SECRET;EXTRA_COMPONENT",
-        ]
-
-        self.mock_course.lti_passports = lti_passports
-
-        expected_log_calls = [
-            mock.call(DaliteXBlock.MALFORMED_LTI_PASSPORT, lti_passports[0]),
-            mock.call(DaliteXBlock.MALFORMED_LTI_PASSPORT, lti_passports[1]),
-        ]
```

Comment:
These tests have been moved here: https://github.com/open-craft/xblock-dalite/pull/2/files#diff-2474661d1dc1672be354883f7e864264R57 . The reasoning is that that all parsing is done in passport_utils so tests in xblocks only check if proper methods from passport_utils are called.